ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ScanConfigClustersTable.tsx

```diff
@@ -10,9 +10,13 @@ import ComplianceClusterStatus from './ComplianceClusterStatus';
 
 type ScanConfigClustersTableProps = {
     clusterScanStatuses: ClusterScanStatus[];
+    headingLevel: 'h2' | 'h3';
 };
 
-function ScanConfigClustersTable({ clusterScanStatuses }: ScanConfigClustersTableProps) {
+function ScanConfigClustersTable({
+    clusterScanStatuses,
+    headingLevel,
+}: ScanConfigClustersTableProps) {
     const [page, setPage] = useState(1);
     const [perPage, setPerPage] = useState(20);
```

Comment:
Also out of scope, but I just realized I missed this line when adding `DEFAULT_COMPLIANCE_PAGE_SIZE`. I feel like this deserves to also be set to the default 10 as well. Thoughts?

Reply:
Yes, agree.